src/applications/personalization/profile-2/components/Profile2Router.jsx

```diff
@@ -55,6 +65,30 @@ class Profile2Router extends Component {
     }
   }
 
+  handleDowntimeApproaching = (downtime, children) => {
+    if (downtime.status === externalServiceStatus.downtimeApproaching) {
+      return (
+        <DowntimeApproaching
+          {...downtime}
+          appTitle="profile"
+          isDowntimeWarningDismissed={this.props.isDowntimeWarningDismissed}
+          dismissDowntimeWarning={this.props.dismissDowntimeWarning}
+          initializeDowntimeWarnings={this.props.initializeDowntimeWarnings}
+          messaging={{
+            title: (
+              <h3>
+                Some parts of the profile will be down for maintenance soon
+              </h3>
+            ),
+          }}
+          className=""
```

Comment:
Yeah :( If we don't set this to a blank string (or some other empty value) then the `row-padded` class will be passed down to `DowntimeNotificationWrapper` that the `DowntimeNotification` renders. There might be a more elegant way to make that `row-padded` class get added optionally while ensuring not to break anything that expects the class to be there.

Reply:
> Yeah :( If we don't set this to a blank string (or some other empty value) then the `row-padded` class will be passed down to `DowntimeNotificationWrapper` that the `DowntimeNotification` renders. There might be a more elegant way to make that `row-padded` class get added optionally while ensuring not to break anything that expects the class to be there.

I think this would be appropriate to include as a code comment.

Reply:
Good call. Done!